src/zino/api/legacy.py

```diff
@@ -29,14 +33,21 @@ def requires_authentication(func: Callable) -> Callable:
 class Zino1BaseServerProtocol(asyncio.Protocol):
     """Base implementation of the Zino 1 protocol, with a basic command dispatcher for subclasses to utilize."""
 
-    def __init__(self, state: Optional[ZinoState] = None, secrets_file: Optional[Union[Path, str]] = "secrets"):
+    def __init__(
+        self,
+        server: Optional["ZinoServer"] = None,
```

Comment:
Are the quotes needed here?

Reply:
.. github missed the line. You have `Optional["ZinoServer"]` with quotes.

Reply:
> Are the quotes needed here?

Apparently, since the symbol is only imported `if typing.TYPE_CHECKING`.  Tests fail if I leave it unquoted.

Reply:
> .. github missed the line. You have `Optional["ZinoServer"]` with quotes.

Huh?

Reply:
Github or my browser or a combo sometimes hides lines.